core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -438,59 +438,60 @@ sealed abstract class Parser1[+A] extends Parser[A] {
     * an epsilon failure, the parsed values (if any) are returned in a
     * list as a successful parse.
     */
-  def rep: Parser[List[A]] =
-    Parser.rep(this)
+  def rep0: Parser0[List[A]] =
```

Comment:
if we are breaking people should we just unify rep0 and repAs here? Maybe in the companion we remove rep0 and just call repAs variants, but leave the methods for convenience?